### ISSUE TITLE
about BatchNorm in your experiments.

### ISSUE THREAD

**Issue by yyhyyh17** [YvanYin/GoalFlow#17]
I noticed that you only use batch size = 2 for the experiment. BatchNorm would perform badly with tiny batch size which leads to training instability and convergence problems. Do you have these problems when training?
Also, I found that you have different norm types in https://github.com/YvanYin/GoalFlow/blob/94161eb422e04b455670cd66db7f26b22b4c2571/vovnet/backbones/norm.py#L9 . Do you tried other normalization methods? If you tried, could you share the results?
Looking forward to your reply, thanks.

**Comment by ZebinX** [comment]
@yyhyyh17 Thanks for the suggestion. Yes, I've also encountered training instability when using just one machine. To address this, the model is currently being trained across four machines, each equipped with 8 GPUs. 
I haven't tried other normalization methods yet.
